<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_6">
     <widget class="QLineEdit" name="lineEdit_info_code">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_infoSong">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>561</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Datos:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_B_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="infoSong_Button">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>601</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Canciones</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_B_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_code_dowload">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="dowload_text">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>471</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargas:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dowload_Button_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_B_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>440</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cliente_dowload">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cliente</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="review_Botton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Calificar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_codigo_R">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>381</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Calificacionn de estrellas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_B_5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>50</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="return_B_1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="info_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Info Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="review_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Review Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="song_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Songs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dowload_Button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>179</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Dowload </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el codigo </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_codigo">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_Button_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>agregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>LOVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Artista</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Artista">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Taylor Swift</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_genero">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>POP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Genero</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Precio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_precio">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>0.99</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_B_4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_duracion">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>3:50</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duracion</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>POP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ROCK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DANCE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>REGGAE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ELECTRONICA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RANCHERA</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
